CMakeLists.txt

```diff
@@ -107,6 +107,11 @@ if (PICOWOTA_WIFI_AP)
 	message("Building in WiFi AP mode.")
 endif()
 
+if (BOOTLOADER_ENTRY_PIN)
```

Comment:
Please use the `picowota_retrieve_variable()` helper, like the other variables, and can we use a similar name? Like `PICOWOTA_ENTRY_PIN`